src/neo4j/_api.py

```diff
@@ -151,9 +151,8 @@ class NotificationDisabledCategory(str, Enum):
     PERFORMANCE = "PERFORMANCE"
     DEPRECATION = "DEPRECATION"
     GENERIC = "GENERIC"
-    #: Requires server version 5.14 or newer.
     SECURITY = "SECURITY"
-    #: Requires server version 5.14 or newer.
+    #: Requires server version 5.13 or newer.
```

Comment:
Is it worth swapping lines 155 and 156?

Reply:
Nope. This is a doc comment, which will be picked up by the docs generation tool. It always refers to what's below.